### DISCUSSION TITLE
Dashed trails and div repulsor

### DISCUSSION THREAD

**Discussion by JusticeMatthew** [matteobruni/tsparticles#4938] [Q&A] [answered]
Hello! I'm back!

Thanks for all the help so far.

Is there a way to create dashed trails with this?

Also for the div repulsor, I have it set and it is behaving how it should, but only on parts of the div (works properly on L, R, and B but for the top, it sinks into the div a bit before being bounced out) 🤔 Are there any height/width props I can manipulate?

**Comment by matteobruni** [comment, marked as answer]
> Hello! I'm back!
> 
> Thanks for all the help so far.
> 
> Is there a way to create dashed trails with this?
> 
> Also for the div repulsor, I have it set and it is behaving how it should, but only on parts of the div (works properly on L, R, and B but for the top, it sinks into the div a bit before being bounced out) 🤔 Are there any height/width props I can manipulate?

You can try with margin and padding, I’m not sure about those values but they’re worth a try.

Dashed trails can’t be achieved at the moment, dotted trails appear when the particles speed is faster than the draw rate, but it’s not intended, it’s only a use limit.

**Reply by JusticeMatthew** [reply]
The margin and padding didn't make an appreciable difference but I was able to achieve the desired result by just hiding a sneaky div in the spot causing issues and gave it the repulse ID

Worked perfectly :)

Thanks again for all you're quick responses, it's a shame your doc website is broken, I hope they figure it out soon

**Reply by matteobruni** [reply]
I’ve checked now and the website is up and running, are you looking at <https://particles.js.org>?

**Reply by JusticeMatthew** [reply]
Oh no, sorry I forgot about that one 😮 
I was referring to the API docs [here](https://particles.js.org/docs/index.html)
I swear I read you saying something about their new system wasn't working with your package or somesuch, idr the exact details 😅

Thanks for all the help!

**Reply by matteobruni** [reply]
Ah yes, TypeDoc 0.24 is close to the release fortunately, so I can have all the docs in one place
